packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js

```diff
@@ -16,6 +16,7 @@ const Translate = {
   'eu-west-3': 'eu-west-3',
   'eu-central-1': 'eu-central-1',
   'eu-north-1': 'eu-north-1',
+  'eu-south-1': 'eu-central-1',
```

Comment:
Translate is not available in eu-south-1. Falling back to eu-central-1

https://docs.aws.amazon.com/general/latest/gr/translate-service.html